src/controls/splash.js

```diff
@@ -17,34 +16,32 @@ const Splash = function Splash(options = {}) {
     url
   } = options;
 
+  const createModal = function createModal(modalContent) {
```

Comment:
suggestion to change param name to content
```
const createModal = function createModal(content) {
````

Reply:
The linter does not approve of this as content is already declared.